doc/module-docs/cc_package_update_upgrade_install/example2.yaml

```diff
@@ -0,0 +1,7 @@
+#cloud-config
+package_update: true
+package_upgrade: true
+snap:
+  commands:
+    00: snap refresh --hold
```

Comment:
should we add a comment or something that indicates that by omitting any duration, `snap refresh --hold` _is_ how you set it to "forever"? For people more familiar with snap this might be obvious, but when i read `To avoid calling `snap refresh` in images with snap installed, setting snap refresh.hold to forever`, my instinct would be to put `snap refresh --hold forever` or something like that. Just a thought. Maybe it's overkill.

Reply:
Thanks @a-dubs fixed the documented examples to be more clear about --hold=forever vs --hold=10m etc. I also updated the command used per @holmanb's review to use `snap get system -d` instead and process the resulting sparse config dict.